src/software/ai/navigator/curve/curve.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+/**
+ * A curve is function that maps time to points
+ */
+
+class Curve
+{
+};
```

Comment:
Could we leave a quick note here about why we have this empty class?

Reply:
ok

Reply:
Done